.buildkite/scripts/buildx.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+set +x
+
+BUILDPLATFORM=${BUILDPLATFORM:-"linux/amd64,linux/arm64"}
+
+#docker run --privileged --rm tonistiigi/binfmt --install all
```

Comment:
I have just copied the script, that is originally used in Jenkins, so that after .ci folder deletion I could simply change the related path. Not sure why original contains the commented code, but I've just kept it for a meantime.